editor/debugger/editor_debugger_node.cpp

```diff
@@ -268,11 +268,14 @@ Error EditorDebuggerNode::start(const String &p_uri) {
 	}
 	stop(true);
 	current_uri = p_uri;
-	if (EDITOR_GET("run/output/always_open_output_on_play")) {
+
+	int action_on_play = EDITOR_GET("run/bottom_panel/action_on_play");
+	if (action_on_play == 1) {
```

Comment:
It would be better to replace magic numbers with some enum (defined in one place), even if EditorSettings have to repeat it.